cloud/amazon/ec2.py

```diff
@@ -1033,6 +1041,9 @@ def create_instances(module, ec2, vpc, override_count=None):
                       private_ip_address = private_ip,
                     ))
 
+                # Spot instances do not support start/stop thereby not having the option to change shutdown behavior
+                params['instance_initiated_shutdown_behavior'] = instance_initiated_shutdown_behavior
```

Comment:
Wouldn't these 3 lines be unnecessary? The comment would still be useful on line 1081.